qiskit/transpiler/__init__.py

```diff
@@ -40,629 +41,787 @@
 Qiskit has four pre-built transpilation pipelines available here:
 :mod:`qiskit.transpiler.preset_passmanagers`.  Unless the reader is familiar with
 quantum circuit optimization methods and their usage, it is best to use one of
-these ready-made routines.
-
+these ready-made routines. By default the preset pass managers into 6 stages:
+
+#. ``init`` - any initial passes that are run before we start embedding the circuit to the backend
+#. ``layout`` - This stage runs layout and maps the virtual qubits in the circuit to the
+   physical qubits on a backend. See :ref:`layout_stage` for more details.
+#. ``routing`` - This stage runs after a layout has been run and will insert any
+   necessary gates to move the qubit states around until it can be run on
+   backend's compuling map. See :ref:`routing_stage` for more details.
+#. ``translation`` - Perform the basis gate translation, in other words translate the gates
+   in the circuit to the target backend's basis set. For more details on this stage you can refer to
+   :ref:`translation_stage`.
+#. ``optimization`` - The main optimization loop, this will typically run in a loop trying to
+   optimize the circuit until a condtion (such as fixed depth) is reached. See
+   :ref:`optimization_stage` for more details.
+#. ``scheduling`` - Any hardware aware scheduling passes. See :ref:`scheduling_stage` for more
+   details.
+
+When using :func:`~.transpile` the specifics of each stage can modified with the ``*_method``
+arguments (e.g. ``layout_method``) and also refer to external plugins. You can refer to
+:mod:`qiskit.transpiler.preset_passmanagers.plugin` for details on this plugin interface.
+
+.. _working_with_preset_pass_managers:
+
+Working With Preset Pass Managers
+=================================
+
+By default Qiskit includes functions to build preset :class:`~.PassManager` objects.
+These preset passmanagers are what get used by the :func:`~.transpile` function
+for each optimization level. There are 4 optimizaiton levels from 0 to 3 where an increasing
+optimization level trades off execution time and computational effort vs potential optimization.
+Where level 0 performs no optimization and just maps the input circuit to the constraints of the
```

Comment:
```suggestion
Level 0 performs no optimization and just maps the input circuit to the constraints of the
```

(otherwise the sentence is just a subordinate clause)